packages/mdc-top-app-bar/_mixins.scss

```diff
@@ -92,5 +91,23 @@
 // Both .mdc-top-app-bar__icon and .mdc-top-app-bar__navigation-icon share all styles except for
 // horizontal padding.
 @mixin mdc-top-app-bar-icon_() {
-  @include mdc-icon-button-base_();
+  @include mdc-ripple-surface;
```

Comment:
Did you revert this with the intent of adopting `mdc-icon-button` within top app bar in a separate PR?

Reply:
Card was previously using icon-toggle, so I went ahead and updated that one. Top app bar doesn't need to change with this PR.